tests/test_lsst_efd_client.py

```diff
@@ -57,6 +61,32 @@ async def make_efd_client():
             await client.drop_database()
 
 
+@contextlib.contextmanager
+def make_synchronous_efd_client():
+    df = pd.read_hdf(PATH / "efd_test.hdf")
+    df1 = pd.read_hdf(PATH / "efd_index_test.hdf", "table")
+    with InfluxDBClient(
+        db="client_test",
+        mode="blocking",
+        output="dataframe",
+    ) as client:
+        client.create_database()
+        client.write(df, measurement="lsst.sal.fooSubSys.test")
+        client.write(df1, measurement="lsst.sal.barSubSys.test")
+        efd_client = EfdClientSync(
+            "test_efd", db_name="client_test", client=client
+        )
+        # Monkey patch the client to point to an existing schema registry
+        # Note this is only available if on the NCSA VPN
+        efd_client.schema_registry = (
+            "https://lsst-schema-registry-efd.ncsa.illinois.edu"
+        )
```

Comment:
One for @rra / @jonathansick / @afausti here as this has clearly been moved from somewhere else in the code, rather than made up by Cristobal, but surely this can't still be doing anything, right? Should we fix this up as part of this work?

Reply:
Also kind of concerning that (assuming that this is indeed no longer functional) that the tests here, which seem to pass, presumably aren't working as intended...